ProcessesApi/V1/Factories/ProcessesSnsFactory.cs

```diff
@@ -65,6 +65,40 @@ public EntityEventSns ProcessClosed(Stateless.StateMachine<string, string>.Trans
             };
         }
 
+        public EntityEventSns ProcessCompleted(Stateless.StateMachine<string, string>.Transition transition, Dictionary<string, object> stateData, Token token)
+        {
```

Comment:
Most of these events are very similar (other than constant names), is there any way to combine / simplify them?